pages/job-category/[JobCategory].js

```diff
@@ -1,18 +1,18 @@
 import { styled } from "styled-components";
```

Comment:
The import is not correct. Please see correct way to do it below. 

```suggestion
import styled from "styled-components";
```

Reply:
This still needs to be fixed